tests/src/test/scala/org/http4s/headers/ForwardedArbitraryInstances.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s.headers
+
+import java.nio.charset.StandardCharsets
+
+import cats.data.NonEmptyList
+import cats.syntax.all._
+import org.http4s.internal.bug
+import org.http4s.laws.discipline.ArbitraryInstances
+import org.http4s.{ParseResult, Uri}
+import org.scalacheck.{Arbitrary, Gen}
+
+private[http4s] trait ForwardedArbitraryInstances
```

Comment:
I wonder, should this trait be moved to `org.http4s.laws.discipline` (where `ArbitraryInstances` sits)?

Reply:
I think yes, but that doesn't need to block this.